content/en/continuous_integration/tests/javascript.md

```diff
@@ -28,20 +28,22 @@ Supported test frameworks:
 * Jest >= 24.8.0
   * Only `jsdom` (in package `jest-environment-jsdom`) and `node` (in package `jest-environment-node`) are supported as test environments. Custom environments like `@jest-runner/electron/environment` in `jest-electron-runner` are not supported.
   * Only [`jest-circus`][1] is supported as [`testRunner`][2].
-  * Jest >= 28 is only supported from `dd-trace>=2.7.0`
+  * Jest >= 28 is only supported from `dd-trace>=2.7.0`.
+  * [`test.concurrent`](#jests-testconcurrent) is not supported.
```

Comment:
Non-blocking, do customers ask about the features that aren't supported? It's not information we like to put in the docs.

Reply:
it's come up in a couple of conversation with customers. Since `test.concurrent` is part of `jest`, without any extra dependency, I'd say it's worth mentioning